client/src/feedbacker-components/comment-panel/comment-panel.js

```diff
@@ -52,6 +52,27 @@ class CommentPanel extends React.Component {
     this.showPanel = this.showPanel.bind(this)
     this.updateCounter = this.updateCounter.bind(this)
     this.updateTagButtonStatus = this.updateTagButtonStatus.bind(this)
+
+    this.interval = null
+  }
+
+  componentDidUpdate() {
+    if (!this.interval) {
+      this.interval = window.setInterval(this.fetchComments, 3000)
+    }
+  }
+
+  componentWillUnmount() {
+    if (this.interval) {
+      window.clearInterval(this.interval)
+      this.interval = null
+    }
+  }
+
+  async fetchComments() {
+    const comments = await apiCall('GET', '/comments')
+    this.props.dispatch(loadComments(comments))
+    this.scrollToBottom()
```

Comment:
Probably shouldn't scroll to bottom every 3 seconds?

Reply:
Need to have two `fetchComments()`, one for posting and one for polling

Reply:
fixed